app/scripts/configImporter.js

```diff
@@ -377,8 +383,11 @@
                             }
                         } else {
                             method = "Set";
-                            entityCopy = extend(true, entity, newUser);
-                            newUser = entityCopy;
+                            // TODO Brett changed up - not working methinks
+                            return requests;
```

Comment:
If you exit the method now you'll not update user groups that can potentially lead to errors during import

Reply:
Yes I know. I tested and it seems to be ok. Tested with multiple users and scenarios.

Reply:
Have you tried to add current user to a rule or report? It seems the import of such entities will fail because there is no user with userId they are related to.
Please also remove string #371 (```method = "Add"```) I can't leave a comment near it.

Reply:
It get's in here if the user exists already. I was testing imports where the user (mine) already existed in the database. Yes I have tested it and it seems to work fine.

Reply:
Nick there is no definition for entity, there is a problem with your recommendation. Please check your response? Or am I missing something?